src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String INDENTATION = "    ";
+    public static final String LINE = INDENTATION
+            + "____________________________________________________________";
+    public static final int MAX_NUM = 100;
+
+    public static Task[] tasks = new Task[MAX_NUM];
+    public static int count = 0;
+    public static Scanner in = new Scanner(System.in);
+
     public static void main(String[] args) {
+        printGreetMsg();
+        String userInput = in.nextLine();
+        processUserInput(userInput);
+        printByeMsg();
+    }
+
+    public static void printGreetMsg() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println(logo);
+        System.out.println(LINE);
+        System.out.println(INDENTATION + "Hello! I'm Duke.");
+        System.out.println(INDENTATION + "What can I do for you?");
+        System.out.println(LINE + "\n");
+    }
+
+    public static void processUserInput(String userInput) {
+        while (!userInput.equalsIgnoreCase("bye")) {
+            String[] inputParts = userInput.split(" ", 2);
+            String command = inputParts[0].toLowerCase();
+            switch (command){
+            case "list":
+                listTasks();
+                break;
+            case "done":
+                doneTask(inputParts[1]);
+                break;
+            default:
+                addTask(command, inputParts[1]);
+                break;
+            }
+            userInput = in.nextLine();
+        }
+    }
+
+    public static void addTask(String taskType, String description) {
+        switch (taskType) {
```

Comment:
I like that the naming you used for this (in particular taskType) is intuitive, despite passing in 'command' in processUserInput.

Reply:
Thank you!